starlight/migrations/0002_auto_20180605_1844.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.0.6 on 2018-06-05 18:44
```

Comment:
It would be a bit cleaner (since this is still all in the same PR) to delete both migrations and to just create one new initial migration. :)